### PR TITLE
librsvg - update to version 2.59.2

### PR DESCRIPTION
**Pull request by fritzkink**
this version is a lot cleaner, no patches needed.

### CODE REVIEW

**Comment by AndWac** [comment]
Have you checked whether gimp still detects it correctly?

File: components/library/librsvg/Makefile [hunk 1]
```diff
@@ -68,18 +56,15 @@ REQUIRED_PACKAGES += developer/vala
 REQUIRED_PACKAGES += library/desktop/gobject/gobject-introspection
 REQUIRED_PACKAGES += system/library/fontconfig
 #needed to build the man page
-PYTHON_REQUIRED_PACKAGES += library/python/docutils
+REQUIRED_PACKAGES += library/python/docutils-39
```
Comment (AndWac) [inline components/library/librsvg/Makefile:59]: Why is this change needed?

File: components/library/librsvg/test/results-all.master [hunk 2]
```diff
@@ -1,23 +1 @@
...
-test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 8 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 5 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 23 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 100 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 16 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
+test result: FAILED. 99 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out;
```
Comment (mtelka) [inline components/library/librsvg/test/results-all.master:1]: What is causing the test failure?  I'd prefer to see all tests passed to avoid surprises.

File: components/library/librsvg/Makefile [hunk 3]
```diff
@@ -43,20 +40,11 @@ CARGO_HOME=             $(@D)/.cargo
 # if -z ignore is added to LD_OPTIONS the build will fail
 LD_OPTIONS=-M /usr/lib/ld/map.noexstk -M /usr/lib/ld/map.noexdata -M /usr/lib/ld/map.pagealign -Bdirect
 
-PATH=                   $(PATH.gnu)
+COMPONENT_PRE_CONFIGURE_ACTION= ( CARGO_HOME=$(CARGO_HOME) cargo install --force cargo-c )
```
Comment (mtelka) [inline components/library/librsvg/Makefile]: I do not like this.  We should package `cargo-c`.

**Comment by fritzkink** [comment]
> Have you checked whether gimp still detects it correctly?

How can I detect?

**Comment by AndWac** [comment]
Install it and build gimp (gmake sample-manifest). Check whether .svg were still build. Typically .svg files will be replaced by .png files. You can also check the logs.

File: components/library/librsvg/test/results-all.master (hunk 2 above)
Comment (fritzkink) [reply components/library/librsvg/test/results-all.master:1]: > What is causing the test failure? I'd prefer to see all tests passed to avoid surprises.

the test result out changed (because build changed to meson framework)
Summary of Failures:

5/5 Rust tests (rsvg_convert)     FAIL           258.24s   exit status 1

Ok:                 4
Expected Fail:      0
Fail:               1
Unexpected Pass:    0
Skipped:            0
Timeout:            0

an this one failed test outputs additionall 
test result: FAILED. 99 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.78s

**Comment by fritzkink** [comment]
> Install it and build gimp (gmake sample-manifest). Check whether .svg were still build. Typically .svg files will be replaced by .png files. You can also check the logs.

but gimp does not link librsvg at all

**Comment by AndWac** [comment]
That might be the case but still with the former 2.59.1 build you provided gimp build wasn't able to ceate svg graphics.

File: components/library/librsvg/test/results-all.master (hunk 2 above)
Comment (mtelka) [reply components/library/librsvg/test/results-all.master:1]: `rsvg_convert`?  That sounds suspicious (maybe related to gimp?).  Could you please try to fix it?

File: components/library/librsvg/Makefile (hunk 3 above)
Comment (mtelka) [reply components/library/librsvg/Makefile]: Here it is: #19418
